### ISSUE TITLE
Puzzles of 【House Of The Lore】

### ISSUE THREAD

**Issue by f0rm2l1n** [shellphish/how2heap#130]
Before talking about my puzzles, I'd like to thank the team shellphish for this wonderful repo! Much appreciate.

After that, I just found the comment in the [house of lore](https://github.com/shellphish/how2heap/blob/master/glibc_2.26/house_of_lore.c) is a bit confusing. 

```
fprintf(stderr, "\nIn the unsorted bin the victim's fwd and bk pointers are nil\n");
fprintf(stderr, "victim->fwd: %p\n", (void *)victim[0]);
fprintf(stderr, "victim->bk: %p\n\n", (void *)victim[1]);
```
The puzzling part is `In the unsorted bin`.

After the debugging, I found that the (nil) pointer only happened when compiled into a 64-bit ELF. That is to say the **fastbin** (128 byte upper bound) is used instead of the unsorted bin as described.

Since the house of the lore is claimed to be tested in a 32-bit architecture, where the 100 byte chunk is indeed out the range of fastbin. However, after the debugging, these two pointers are by no means nil in the unsorted bin double linked list...

To summarize, I think the comment of  `pointers are nil` should be removed.

**Comment by Kyle-Kyle** [comment]
Nice catch. In fact, I found the problem too. The updated version is in `refactor` branch
